src/Microsoft.Azure.SignalR.Common/Endpoints/AadAccessKey.cs

```diff
@@ -23,15 +24,17 @@ internal class AadAccessKey : AccessKey, IDisposable
 
         private int initialized = 0;
 
-        private readonly TaskCompletionSource<bool> _authorizeTcs = new TaskCompletionSource<bool>();
+        private readonly TaskCompletionSource<bool> _initializedTcs = new TaskCompletionSource<bool>();
```

Comment:
use `new TaskCompletionSource<bool>(TaskCreationOptions.RunContinuationsAsynchronously)`

Reply:
nit: looks like `<object>` and `TrySetResult(null)` is enough